rtmr/extend/extend.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package extend
```

Comment:
I think this should be moved to go-configfs-tsm with a more generalized API for expected PCR->RTMR mapping. Every function appears to be exported. You do really need everything to be public? Are library users expected to use all these functions, or is there an easier path to follow that we can export and keep the rest internal until a need arises to export them?

Reply:
I am OK to move this package to go-configfs-tsm. Do you have any thoughts on where we should store the TDX specific PCR->RTMR mappings.